### PR TITLE
fix readonly struct ctor

### PR DESCRIPTION
**Pull request by sungam3r**
@danielmarbach We need beta-4 :)

### CODE REVIEW

**Comment by sungam3r** [comment]
image

File: src/PublicApiGenerator/CodeNormalizer.cs [hunk 1]
```diff
@@ -51,6 +51,7 @@ public static string NormalizeGeneratedCode(StringWriter writer)
 
             gennedClass = gennedClass.Replace("class " + StaticMarker, "static class ");
```
Comment (danielmarbach) [inline src/PublicApiGenerator/CodeNormalizer.cs]: Should we also make sure to remove the static marker?

File: src/PublicApiGeneratorTests/Struct_readonly.cs [hunk 2]
```diff
@@ -12,6 +12,19 @@ public void Should_output()
 @"namespace PublicApiGeneratorTests.Examples
 {
     public readonly struct ReadonlyStruct { }
+}");
+        }
+
+        [Fact]
+        public void Should_output_with_ctor()
```
Comment (danielmarbach) [inline src/PublicApiGeneratorTests/Struct_readonly.cs]: Could you also add a similar test for a class with a static constructor?

File: src/PublicApiGenerator/CodeNormalizer.cs (hunk 1 above)
Comment (danielmarbach) [reply src/PublicApiGenerator/CodeNormalizer.cs]: and maybe while we are at it also remove `AttributeMarker `

Comment (sungam3r) [reply src/PublicApiGenerator/CodeNormalizer.cs]: static ctor is not a part of public api, I added tests

Comment (sungam3r) [reply src/PublicApiGenerator/CodeNormalizer.cs]: > and maybe while we are at it also remove AttributeMarker

Let's remove it only if/when we see such a situation.

File: src/PublicApiGeneratorTests/Struct_readonly.cs (hunk 2 above)
Comment (sungam3r) [reply src/PublicApiGeneratorTests/Struct_readonly.cs]: done

**Comment by sungam3r** [comment]
@danielmarbach Could you please publish beta-4?

**Comment by danielmarbach** [comment]
Certainly not ;)

**Comment by danielmarbach** [comment]
It's up. Sorry for the delay

**Pull request merged**